python/ray/worker.py

```diff
@@ -1641,6 +1641,29 @@ def wait(object_ids, num_returns=1, timeout=None):
         return ready_ids, remaining_ids
 
 
+def kill(actor):
+    """Kill an actor forcefully.
+
+    This will interrupt any running tasks on the actor, causing them to fail
+    immediately. Any atexit handlers installed in the actor will still be run.
+
+    If you want to kill the actor but let pending tasks finish,
+    you can call ``actor.__ray_terminate__.remote()`` instead to queue a
```

Comment:
Egh, can we also rename `__ray_terminate__`? This is really not a good idea to use double underscores. Those are reserved for Python use.

Reply:
Maybe the API should be `ray.terminate(actor)` and `ray.kill(actor)`. It is OK then for `ray.terminate` to call `actor.__ray_terminate__` and for subclasses of actors to subclass `__ray_terminate__` to customize its behavior. That would be aligned with Python best practices. But public API should never call `__ray_terminate__` directly.

Reply:
Yes, but let's discuss/do that separately.

Reply:
I opened #7382 to followup here.